Backend/ServiceLayer/Services/ResetService.cs

```diff
@@ -0,0 +1,229 @@
+            string userlastName = "";
+            string userFullName = userFirstName + " " + userlastName;
+            string template = "Hi {0}, \r\n" +
+                                             "You recently requested to reset your password for your KFC account, click the link below to reset it.\r\n" +
+                                             "The URL is only valid for the next 5 minutes\r\n {1}" +
+                                             "If you did not request to reset your password, please contact us by responding to this email.\r\n\r\n" +
+                                             "Thanks, KFC Team";
+            string data = "userFirstName, resetURL";
+            string resetPasswordBodyString = string.Format(template, data);
+            //Create the email service object
+            EmailService es = new EmailService();
+            //Create the message that will be sent
+            MimeMessage emailToSend = es.createEmailPlainBody(userFullName, receiverEmail, resetPasswordSubjectString, resetPasswordBodyString);
+            //Send the email with the message
+            es.sendEmail(emailToSend);
+            
+        }
+
+        //Function to create the email is user doesn't exist
+        public void SendResetEmailUserDoesNotExist(string receiverEmail)
+        {
+            string userFullName = "Unknown";
+            string resetPasswordUserDoesNotExistEmailBody = "Hello, \r\n" +
+                              "You (or someone else) entered this email address when trying to reset the password of a KFC account.\r\n" +
+                              "However, this email address is not on our database of registered users and therefore the attempt to reset the password has failed.\r\n" +
+                              "If you have a KFC account and were expecting this email, please try again using the email address you gave when opening your account." +
+                              "If you do not have a KFC account, please ignore this email.\r\n" +
+                              "For more information about KFC, please visit www.kfc.com/faq \r\n\r\n" +
+                              "Best Regards, KFC Team";
+            //Create the email service object
+            EmailService es = new EmailService();
+            //Create the message that will be sent
+            MimeMessage emailToSend = es.createEmailPlainBody(userFullName, receiverEmail, resetPasswordSubjectString, resetPasswordUserDoesNotExistEmailBody);
+            //Send the email with the message
+            es.sendEmail(emailToSend);
+        }
+
+        //Function to create the email if the password was changed
+        public void SendPasswordChange(string receiverEmail)
+        {
+            //Need SQL Query to get info about user from DB
+            string userFirstName = "";
+            string userlastName = "";
+            string userFullName = userFirstName + " " + userlastName;
+            string template = "Hi {0}, \r\n" +
```

Comment:
Each of these templates should probably be located in a separate email template service or such. Whatever, this isn't that important.